starter-code/index.js

```diff
@@ -1,10 +1,57 @@
-var SortedList = function() {};
+var SortedList = function() {
+	this.items = [];
+	this.length = 0;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var sortedList = new SortedList();
```

Comment:
I don't think this line serves any purpose here `var sortedList = new SortedList();`